tuf/api/metadata.py

```diff
@@ -776,7 +778,7 @@ def verify_signature(
             SerializationError,
         ) as e:
             # Log unexpected failure, but continue as if there was no signature
-            logger.info("Key %s failed to verify sig: %s", self.keyid, str(e))
+            logger.debug("Key %s failed to verify sig: %s", self.keyid, str(e))
```

Comment:
That's not actually the warning I commented about : this one I'd be fine as a warning -- here the situation could be a configuration error or a bug so a warning seems reasonable... but the Metadata as a whole may still be verified so we just raise UnsignedMetadataError for this specific sig so that verify_delegate can handle it.